### PR TITLE
[Issue-1638] TP - iso.dhcp needs a default value (no) for iso generation

### LINKED ISSUE #1638: Traffic Portal - Gen Iso: Copy server attributes no longer works
In Traffic Portal if you attempt to generate an ISO by copying the attributes of a server from the server table, the portal will not fill in the fields for you. Instead all of the required fields will be red and you will have to manually enter the data.

This functionality used to work, so I am not sure what broke.

### PR DESCRIPTION
**Pull request by mitchell852**
fixes #1638

### CODE REVIEW

**Comment by dneuman64** [comment]
This allows the portal to fill out the fields based on the server you choose?  I will have to test that

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/765/
Test PASSed.

**Comment by mitchell852** [comment]
yes @dneuman64 - i removed that default (dhcp) which resulted in a null pointer which killed the data binding needed to fill out the fields based on the server you choose.

File: traffic_portal/app/src/common/modules/form/iso/FormISOController.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ var FormISOController = function(servers, osversions, $scope, $anchorScroll, for
 		{ value: 'no', label: 'no' }
 	];
 
+	$scope.iso = {
+		dhcp: 'no'
+	};
+
 	$scope.isDHCP = function() {
 		return $scope.iso.dhcp == 'yes';
```
Comment (mitchell852) [inline traffic_portal/app/src/common/modules/form/iso/FormISOController.js]: this line was null pointering due to the removal of lines 33-35

**Pull request merged**